src/shared/util.js

```diff
@@ -1130,6 +1130,19 @@ if (
   };
 }
 
+// TODO: Remove this once the `javascript.options.experimental.math_sumprecise`
```

Comment:
Given the browser compatibility table at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/sumPrecise#browser_compatibility should we extend this to say that it can only be removed once the feature landed in the browser versions we support?

Reply:
These sort of feature rollout prefs are usually not removed until [the proposal](https://github.com/tc39/proposal-math-sum/) reaches stage 4, at which point it should be implemented in all browsers.
In older browsers we should, at that point, be able to rely on a core-js polyfill.